docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -217,6 +234,12 @@ created by using a `$name.volume` quadlet file.
 
 This key can be listed multiple times.
 
+#### `Network=`
+
+Specify a custom network for the container. This has the same format as the `--network` option
+to `podman run`. For example, use `host` to use the host network in the container, or `none` to
+not set up networking in the container.
```

Comment:
You can set --network multiple times when you specify network names. I think you should allow that too.

Reply:
Added in a separate commit